eo-maven-plugin/src/test/java/org/eolang/maven/EOplugin.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2016-2025 Objectionary.com
+ * SPDX-License-Identifier: MIT
+ */
+package org.eolang.maven;
+
+import com.jcabi.manifests.Manifests;
+import com.yegor256.farea.Farea;
+import com.yegor256.farea.Plugin;
+import java.io.IOException;
+
+/**
+ * Configures the EO Maven plugin within a {@link Farea}.
+ * @since 0.52
+ */
+final class EOplugin {
```

Comment:
@Suban05 let's call it `AppendedPlugin` and make it implement `org.cactoos.Scalar<com.yegor256.farea.Execution>` so we could do:
```java
new AppendedPlugin(f).get().configuration(...)...
```